internal/graph/extensions/live_query/runtime/runtime.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Upbound Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package runtime
+
+import (
+	"context"
+	"sync/atomic"
+
+	"github.com/99designs/gqlgen/graphql"
+	"github.com/vektah/gqlparser/v2/ast"
+)
+
+type QueryResolveFn func(ctx context.Context, sel ast.SelectionSet) graphql.Marshaler
+type CancelFn func()
+
+type LiveQuery struct {
```

Comment:
Would it be possible to add a comment block or two explaining why we need this/what it's used for?

Reply:
let me know if the comments I added are sufficient.